.circleci/config.yml

```diff
@@ -497,6 +499,7 @@ workflows:
           airflow_version: 1.10.10.dev
           distribution_name: alpine3.10
           docker_repo: "astronomerio/ap-airflow"
+          extra_args: "--build-arg VERSION='=='$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-1.10.10.build)"
```

Comment:
You edited this manually, and didn't update the templates/the rest to match.

Reply:
Nope, didn't edit manually

Reply:
I added conditions in the template file to only use this for DEV version